scripts/ci/dockerfiles/polkadot_injected_release.Dockerfile

```diff
@@ -44,6 +44,8 @@ USER polkadot
 
 # check if executable works in this container
 RUN /usr/bin/polkadot --version
+RUN /usr/bin/polkadot-execute-worker --version
+RUN /usr/bin/polkadot-prepare-worker --version
```

Comment:
@paritytech/ci I want to double-check that this file got reviewed, to make sure it works with multiple binaries, as it wasn't part of this PR diff before today.

Reply:
It's not gonna work /yet/ given there aren't any releases with the separate binaries so far, but lgtm.